build.tmpl.yml

```diff
@@ -228,66 +225,51 @@ archives:
   - id: windows-cgo
     builds:
       - windows-cgo
-    replacements:
-      darwin: macOS
-      386: 32bit
-      amd64: 64bit
     format_overrides:
       - goos: windows
         format: zip
-    name_template: "{{ .ProjectName }}_{{ .Version }}-{{ .Os }}_sqlite_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}"
+    name_template: >-
+      {{ .ProjectName }}_{{ .Version }}-windows_sqlite_{{- if eq .Arch "amd64" }}64bit{{- else if eq .Arch "386" }}32bit{{- else }}{{ .Arch }}{{ end }}
 
   - id: macos-cgo
     builds:
       - macos-cgo
-    replacements:
-      darwin: macOS
-      386: 32bit
-      amd64: 64bit
     format_overrides:
       - goos: windows
         format: zip
-    name_template: "{{ .ProjectName }}_{{ .Version }}-{{ .Os }}_sqlite_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}"
+    name_template: >-
+      {{ .ProjectName }}_{{ .Version }}-macOS_sqlite{{- if eq .Arch "amd64" }}64bit{{- else if eq .Arch "386" }}32bit{{- else }}{{ .Arch }}{{ end }}
 
   - id: linux-cgo
     builds:
       - linux-cgo-arm64
       - linux-cgo-amd64
       - linux-cgo-arm
-    replacements:
-      darwin: macOS
-      386: 32bit
-      amd64: 64bit
     format_overrides:
       - goos: windows
         format: zip
-    name_template: "{{ .ProjectName }}_{{ .Version }}-{{ .Os }}_sqlite_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}"
+    name_template: >-
+      {{ .ProjectName }}_{{ .Version }}-linux_sqlite{{- if eq .Arch "amd64" }}64bit{{- else if eq .Arch "386" }}32bit{{- else }}{{ .Arch }}{{ end }}
 
   - id: alpine-cgo
     builds:
       - alpine-cgo-amd64
       - alpine-cgo-arm64
       - alpine-cgo-arm
-    replacements:
-      darwin: macOS
-      386: 32bit
-      amd64: 64bit
     format_overrides:
       - goos: windows
         format: zip
-    name_template: "{{ .ProjectName }}_{{ .Version }}-{{ .Os }}_sqlite_libmusl_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}"
+    name_template: >-
+      {{ .ProjectName }}_{{ .Version }}-linux_sqlite_libmusl{{- if eq .Arch "amd64" }}64bit{{- else if eq .Arch "386" }}32bit{{- else }}{{ .Arch }}{{ end }}
 
   - id: default
     builds:
       - default
-    replacements:
-      darwin: macOS
-      386: 32bit
-      amd64: 64bit
     format_overrides:
       - goos: windows
         format: zip
-    name_template: "{{ .ProjectName }}_{{ .Version }}-{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}"
+    name_template: >-
+      {{ .ProjectName }}_{{ .Version }}-{{- if eq .Os "darwin" }}macOS{{- else }}{{ .Os }}{{ end }}_{{- if eq .Arch "amd64" }}64bit{{- else if eq .Arch "386" }}32bit{{- else }}{{ .Arch }}{{ end }}
```

Comment:
I believe that `{{-` strips the preceding newline, which you can use to make this multiline which might make it easier to read